aws/lambda/hud-query-proxy/lambda_function.py

```diff
@@ -46,7 +46,7 @@ async def run_query(query: str, params: Any = None) -> List[Dict[str, Any]]:
 
 PR_RE = re.compile(r"(.*)\(#([0-9]+)\)$")
 NUM_COMMITS = int(os.getenv("num_commits", 100))
-BRANCHES = os.getenv("branches", "master,release/1.9,nightly").split(",")
+BRANCHES = os.getenv("branches", "master,main,release/1.9,nightly").split(",")
```

Comment:
note to self: p sure this also isn't used anymore

Reply:
@janeyx99 Do you mean the whole lambda isn't used anymore?

Reply:
For now you can still merge these changes in (the deletions shouldn't come rn anyway), but I wanted to note these for later action